static/js/recent_topics.js

```diff
@@ -663,8 +674,9 @@ export function change_focused_element(e, input_key) {
         set_table_focus(row_focus, col_focus);
         return true;
     }
-    if (current_focus_elem) {
+    if (current_focus_elem && input_key !== "escape") {
         current_focus_elem.trigger("focus");
+        return true;
```

Comment:
This has made it impossible to type an actual ‘t’ in the recent topics search bar.

Reply:
Fixed! Thanks!